config/webpack.common.js

```diff
@@ -189,7 +189,7 @@ module.exports = function (options) {
             }, {
               loader: 'css-loader',
               options: {
-                minimize: isProd,
+                minimize: true,
```

Comment:
it doesn't make a difference though in terms of inspecting the code - the inspectors pull everything up as they should. The point for doing this is so that the dev environment can mimic production - I found a few bugs by changing this to always minimized.